src/AElf.Kernel.TransactionPool/Infrastructure/TxHub.cs

```diff
@@ -94,6 +94,7 @@ public Task<TransactionReceipt> GetTransactionReceiptAsync(Hash transactionId)
                 collection.TryAdd(receipt.Transaction.RefBlockNumber, receipts);
             }
 
+            if (receipts.Keys.Contains(receipt.TransactionId)) return;
```

Comment:
use dict.ContainsKey instead of dict.Keys.Contains

Reply:
updated